lib/net/pktdrv/pktdrv.h

```diff
@@ -1,6 +1,9 @@
 #ifndef _Pktdrv_
 #define _Pktdrv_
 
+extern void* PktdrvMmioBase;
+extern unsigned int PktdrvInterrupt;

```

Comment:
This is a hack, so xbe-loader can relocate the NIC (which implies changing the MMIO addresses).

If this was only to get rid of the macros, these variables wouldn't have to be exposed outside (they'd be `static const`). I'm a bit torn about this implementation as there's probably better ways to handle this (such as passing an address to `Pktdrv_Init` or something - much like it's being done for kernel function `AvSendTVEncoderOption`, which expects the GPU MMIO address as first argument).

Reply:
I would prefer we had a function that wrapped these variables for setting, but that can happen in a follow up.

Reply:
For the record: A similar issue exists for USB.